Sources/TransferMethod/CountryCurrencyCell.swift

```diff
@@ -59,7 +59,7 @@ final class CountryCurrencyCell: GenericCell<CountryCurrencyCellConfiguration> {
 
             valueLabel.text = accessoryType == .checkmark ? "" : configuration.value
             valueLabel.accessibilityLabel = configuration.value
-            valueLabel.accessibilityIdentifier = configuration.value
+            valueLabel.accessibilityIdentifier = "\(configuration.title)_value"
```

Comment:
Let's not make this change for now. keep it the way it was

Reply:
fixed